flow/Trace.cpp

```diff
@@ -365,6 +366,11 @@ struct TraceLog {
 		if (r.rolesString.size() > 0) {
 			fields.addField("Roles", r.rolesString);
 		}
+
+		for (auto field : universalFields) {
```

Comment:
Should probably by `const auto&`. We could also do the fancy new `const auto& [first, second] : universalFields` syntax